learntools/machine_learning/nbs/tut2.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "5f1611b2-6aae-4694-9516-d7651880b314",
+    "_uuid": "8c8591f89df1d36843546a8241150b966682d4c8"
+   },
+   "source": [
+    "**[Machine Learning Course Home Page](kaggle.com/learn/machine-learning).**\n",
+    "\n",
+    "\n",
+    "---\n",
+    "\n",
+    "# Starting Your Project\n",
+    "\n",
+    "You are about to build a simple model and then continually improve it.\n",
+    "\n",
+    "\n",
+    "---\n",
+    "# Using Pandas to Get Familiar With Your Data\n",
+    "\n",
+    "The first thing you'll want to do is familiarize yourself with the data.  You'll use the Pandas library for this.  Pandas is the primary tool that modern data scientists use for exploring and manipulating data.  Most people abbreviate pandas in their code as `pd`.  We do this with the command"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_cell_guid": "1dde37af-64d3-45fc-b1b9-c300a1d49fc2",
+    "_uuid": "0d9fdf505bc0a0c98d4f3d8bb5ef387f05cda426",
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "b62d4ca0-38c8-4cbf-91f8-d5b95eb293fc",
+    "_uuid": "390958537e6e6f663708b2d2cd4bd42b3a3d4cfd"
+   },
+   "source": [
+    "The most important part of the Pandas library is the DataFrame.  A DataFrame holds the type of data you might think of as a table. This is similar to a sheet in Excel, or a table in a SQL database. The Pandas DataFrame has powerful methods for most things you'll want to do with this type of data.  Let's start by looking at a basic data overview with our example data from Melbourne and the data you'll be working with from Iowa.\n",
+    "\n",
+    "The example will use data at the file path **`../input/melbourne-housing-snapshot/melb_data.csv`**.  Your data will be available in your notebook at **`../input/house-prices-advanced-regression-techniques/train.csv`** (which is already typed into the sample code for you). **Remember that you will be using the Iowa data instead of the Melbourne data when writing your own code.**\n",
```

Comment:
> Remember that you will be using the Iowa data instead of the Melbourne data when writing your own code.

Isn't this the first time that Melbourne and Iowa are mentioned? If so, it seems odd to use the word "remember".

Maybe before talking about Pandas you could have a brief section at the top that frames the problem. Recapping the scenario of wanting to predict housing prices (in order to buy low and sell high), then introducing the two datasets we'll be using, and the fact that the example code will use Melbourne, and the code they write will use Iowa (insert in-universe explanation here).

Reply:
You might even defer talking about the Iowa data until they're actually in the Exercise notebook. (I definitely wouldn't get as specific as talking about details like the Iowa file path at this point).